docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/bias_quantification.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #66.        return paf_bias, O_bias
</pre><p>Is PAF bias ever of primary interest to us? I can't think of when it would be. I'd favor switching this to the absolute and relative biases as in the other scenarios.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Hmm, I was thinking it was worthwhile to include the PAF bias simply to show that it is often different than the cases averted bias to encourage not to stopping bias assessment just with the PAF. Do you agree?</p><p> </p><p>In terms of absolute/relative bias, I had a hard time thinking through the best way to present this here. I'm currently presenting absolute bias (difference in cases averted due to intervention) normalized to the estimate of cases averted while considering correlation. </p><p> </p><p>I've been thinking of the relative bias as the difference in the population outcome rate in the intervention scenario as a proportion of the outcome rate without the intervention, which doesn't really seem like something that we would typically measure.</p><p> </p><p>Either way, I think I should add some description to what is actually being measured here. Do you have thoughts on whether we should be evaluating the relative scale I described - or do you have alternative ideas on how to estimate bias on the relative scale that I am missing?</p>

Reply:
> to encourage not to stopping bias assessment just with the PAF

I see your point, but I think this could just be a note (which I believe you already have) not to only look at PAF bias. I also think a similar note could be added to the main risk/risk correlation page, or maybe it should just be revised to show how the outcome is biased and not only the PAF. When I wrote it I was thinking more binary (is this biased or not?) which _is_ the same between PAF and outcome, and not about magnitude, which is _not_ the same.

> I've been thinking of the relative bias as the difference in the population outcome rate in the intervention scenario as a proportion of the outcome rate without the intervention, **which doesn't really seem like something that we would typically measure.**

I feel like I have seen plots of e.g. % of SAM/MAM cases averted -- is that not the same thing? But if it's really true that we don't care about relative bias, then maybe we don't need to show that either! The less things we plot, the better IMO -- it makes our story clearer.